packages/runtime/src/next/components/tests/controls/checkbox-control.test.tsx

```diff
@@ -74,14 +74,14 @@ describe('Page', () => {
     )
 
     expect(snapshot).toMatchSnapshot()
-    expect(screen.getByTestId(testId)).toHaveTextContent('alpha')
+    expect(screen.getByTestId(testId).textContent).toBe(String(bool))
```

Comment:
Don't have to change it, but I think a more generic way to assert here would be something like `expect(JSON.parse(screen.getByTestId(testId).textContent)).toBe(bool)`

Reply:
More generic == something that would work universally for all control types.

Reply:
You're right that it's more generic, but I decided to intentionally avoid doing any operations on the DOM content to prevent any kind of unintentional transformation. Obviously that won't matter here since the boolean is a trivial case, it's more of a general principle thing.

Reply:
Non-trivial cases is actually why I brought this up in the first place. Ad hoc testing for a boolean is easy, but for other controls with object values we will have to parse the text content back -- which is totally fine. `JSON.parse` is the exact reverse of `stringify`, and will throw if there is anything extra in the `textContent` (other than trailing whitespace).